src/components/Jobs/Video/Video.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useRef } from 'react'
```

Comment:
Receiving `DOMException: The play() request was interrupted` when video tries to load breaking the site, locally. Try to await the video to load before trying to play it. https://developers.google.com/web/updates/2017/06/play-request-was-interrupted

Reply:
captured promise and on error added pause